CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# __2.14.6__
```

Comment:
This changelog is actually auto generated. You can add a new entry by running the `scripts/new-change` script and following the instructions, and then commit the new file created by the script in .changes/next-release with your changes.

Reply:
My bad, I didn't even fully read a paragraph regarding this script in contribution guide since I thought that this is optional... Fixed this and slightly modified the guide itself to highlight the fact that script usage is mandatory (hopefully it's OK to do this in this PR and not in the scope of a separate issue).